app/models/bot/tagger.rb

```diff
@@ -0,0 +1,65 @@
+require 'json'
+
+class Bot::Tagger < BotUser
+  check_settings
+  class Error < ::StandardError
+  end
+
+  def self.get_tag_text(tag_id,auto_tag_prefix,ignore_autotags)
+    tag=TagText.find_by_id(tag_id).text
+    if tag.nil? || (ignore_autotags && tag[0]==auto_tag_prefix)
```

Comment:
**question** is the intention here to return nil if the tag can't be found? line 9 will raise an exception, skipping this and sending us to the rescue StandardError, if the tag isn't found. if you'd prefer to return nil without exception you can do `tag = TagText.find_by_id(tag_id)&.text` - the important part here being the safe operator (& before method call)

Reply:
Agreed. Implemented.